ui/html/limit-reached.html

```diff
@@ -16,7 +16,7 @@ <h3 class="js-i18n sm-margin-bottom">We're all out of coins!</h3>
     <button
       id="limit-reached-ok"
       display="inline-block"
-      class="filled-progress-button button ok js-i18n"
+      class="filled-progress-button button ok"
```

Comment:
The js-i18n should be on the inner div, here we're not translating this button at all.